jgrapht-ext/pom.xml

```diff
@@ -61,6 +61,9 @@
 					<archive>
 						<manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
 					</archive>
+					<excludes>
+						<exclude>**/package.html</exclude>
+					</excludes>
```

Comment:
I will remove this, as it does not seem to be the correct solution. 

The problem I am experiencing is that the package.html file (javadoc description for package) sometimes ends up in the jar, but I am starting to guess that this is Eclipse related.